pypeit/reduce.py

```diff
@@ -994,6 +1005,9 @@ def find_objects_pypeline(self, image, std_trace=None,
             boxcar_rad_skymask = self.par['reduce']['extraction']['boxcar_radius'] / self.get_platescale(None),
         else:
             boxcar_rad_skymask = None
+        # user provided optimal fwhm in pixels
```

Comment:
See my comment above, this should simply be a boolean that you pass to object finding.

Reply:
Done